modules/ROOT/pages/index.adoc

```diff
@@ -12,8 +12,10 @@ The following table provides component information for Anypoint Platform PCE:
 [%header%autowidth.spread]
 |===
 | Component | Included | Not Included | Description
-| Anypoint Runtime Manager | X | | Enables you to track and manage your deployed Mule applications
+| Anypoint Runtime Manager | X |  | Enables you to track and manage your deployed Mule applications.
+| Runtime Fabric Management (including Security Edge and Tokenization) |  | X | Automates the deployment and orchestration of customer-hosted Mule applications and API gateways.
```

Comment:
Runtime Fabric Management is not a part of PCE, because it only supports RTF (for hybrid deployment) and RTF-cloud(for cloud version )

Reply:
Right, this is not included.

Reply:
Thank you! It's marked as not included.